### ISSUE TITLE
Add reference to `list_all_templates()` in `use_ad_template()` documentation

### ISSUE THREAD

**Issue by thomas-neitmann** [Roche-GSK/admiral#698]
This should make it clear for user how to discover which templates are available.

**Comment by hamzarahal** [comment]
@thomas-neitmann  it should be a kind of simple description on what list_all_templates() is doing #' Diplay which templates are available] coming before function tags and no impact on those ones, otherwise no need to update the test file?

**Comment by thomas-neitmann** [comment]
Yes just updates to deocumentation. No need to touch any test files.
